metrics/scripts/grafana-provisioning/dashboards/testnet-monitor.json

```diff
@@ -5789,7 +5789,7 @@
           "measurement": "cluster_info-vote-count",
           "orderByTime": "ASC",
           "policy": "autogen",
-          "query": "SELECT mean(\"count\")  AS \"serialize_bank_rc\" FROM \"$testnet\".\"autogen\".\"bank-rc-serialize-ms\" WHERE host_id::tag =~ /$hostid/ AND $timeFilter GROUP BY time($__interval)",
+          "query": "SELECT mean(\"count\")  AS \"serialize_bank\" FROM \"$testnet\".\"autogen\".\"bank-serialize-ms\" WHERE host_id::tag =~ /$hostid/ AND $timeFilter GROUP BY time($__interval)",
```

Comment:
New name is constructed at: https://github.com/solana-labs/solana/pull/7182/files#diff-0f0af8fcfa1444e1b07d671c5bfa4a69R213